crucible/crucible.cabal

```diff
@@ -67,6 +67,7 @@ library
     Lang.Crucible.Analysis.Fixpoint.Components
     Lang.Crucible.Analysis.Postdom
     Lang.Crucible.Analysis.Reachable
+    Lang.Crucible.Analysis.Taint
```

Comment:
See the comment in the module - I don't think this is actually used anywhere.

Reply:
Removed.